internal/cli/cmd/cluster/dataprotection.go

```diff
@@ -75,26 +75,18 @@ var (
 		# edit backup policy
 		kbcli cluster edit-backup-policy <backup-policy-name>
 
-        # enable pitr
-		kbcli cluster edit-backup-policy <backup-policy-name> --set schedule.logfile.enable=true
+        # update backup Repo
+		kbcli cluster edit-backup-policy <backup-policy-name> --set backupRepoName=<backup-repo-name>
 
 	    # using short cmd to edit backup policy
         kbcli cluster edit-bp <backup-policy-name>
 	`)
 	createBackupExample = templates.Examples(`
-		# create a backup, the default type is snapshot.
-		kbcli cluster backup mycluster
-
-		# create a snapshot backup
-		# create a snapshot of the cluster's persistent volume for backup
-		kbcli cluster backup mycluster --type snapshot
-
-		# create a datafile backup
-		# backup all files under the data directory and save them to the specified storage, only full backup is supported now.
-		kbcli cluster backup mycluster --type datafile 
+		# create a backup with method 
```

Comment:
```
# create a backup with a specified method, run "kbcli cluster desc-backup-policy mycluster" to show the cluster supported backup methods
```

Reply:
And now cli support `kbcli cluster desc-backup-policy BACKUP-POLICY-NAME`, prefer to `kbcli cluster desc-backup-policy CLUSTE-NAME --name=BACKUP-POLICY-NAME`, if the `--name` is not specified, describe the default backup policy.

Reply:
ok, I will change cmd `kbcli cluster desc-backup-policy`, and add cmd `kbcli dp desc-backup-policy`.